<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtnMonodroga</class>
 <widget class="QWidget" name="vtnMonodroga">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alta Monodroga</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>* Tipo de Venta</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineNombre">
         <property name="statusTip">
          <string>Ingrese el nombre de la monodroga</string>
         </property>
         <property name="accessibleDescription">
          <string>monodroga</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBuscar">
         <property name="text">
          <string>Buscar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>* Nombre</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Descripción</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="cmbTipoVenta">
         <property name="statusTip">
          <string>Seleccione el tipo de venta que la monodroga exige</string>
         </property>
         <item>
          <property name="text">
           <string>Libre</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Receta</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Receta Archivada</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="txtDescripcion">
       <property name="statusTip">
        <string>Ingrese la descripción de la monodroga (opcional)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupMonodroga">
     <property name="title">
      <string>Monodroga</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tablaMonodroga">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Nombre</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo de Venta</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Descripción</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineNombre</tabstop>
  <tabstop>btnBuscar</tabstop>
  <tabstop>cmbTipoVenta</tabstop>
  <tabstop>txtDescripcion</tabstop>
  <tabstop>tablaMonodroga</tabstop>
  <tabstop>btnAceptar</tabstop>
  <tabstop>btnCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
